database/meta.go

```diff
@@ -178,8 +140,9 @@ func (m *MetaDB) LastCheck() (time.Time, semver.Version) {
 	return time.Time{}, semver.Version{}
 }
 
-// AddNewDB adds a new database to the DBMetaInfo table
-func (m *MetaDB) AddNewDB(name string) error {
+// AddNewDB adds a new database to the DBMetaInfo table. All new databases are
+// ready to be rolling databases.
+func (m *MetaDB) AddNewDB(name string, currentChunk, totalChunks int) error {
```

Comment:
This was changed to put all the pieces necessary for a rolling database in right away. That way when converting to a rolling database later all we have to do is toggle the rolling flag from false to true.